### PR TITLE
IsImported() check method in __init__.py file

### PR DESCRIPTION
**Pull request by rubenzorrilla**
Addition of a CheckImportedApplication() to check if applications have been imported in the MainScript. As a sample of its use, it has been implemented in the FluidDynamicsApplication solvers.

### CODE REVIEW

**Comment by loumalouomega** [comment]
@KratosMultiphysics/structural-mechanics We should so something equivalent

**Comment by philbucher** [comment]
@loumalouomega are you doing it? Or shall I?

**Comment by loumalouomega** [comment]
As you wish

**Comment by philbucher** [comment]
ok then I do it since you did #1192 :)

**Comment by loumalouomega** [comment]
:D

**Comment by rubenzorrilla** [comment]
@pooyan-dadvand @jcotela as we discuss yesterday, I've renamed CheckImportedApplications() to CheckRegisteredApplications().

File: kratos/python_scripts/linear_solver_factory.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 from KratosMultiphysics import *
 
-CheckImportedApplications("ExternalSolversApplication")
+CheckRegisteredApplications("ExternalSolversApplication")
```
Comment (loumalouomega) [inline kratos/python_scripts/linear_solver_factory.py]: @philbucher PING

Comment (philbucher) [reply kratos/python_scripts/linear_solver_factory.py]: Done, pls reapprove :)

**Review by jcotela — APPROVED** [review]

**Pull request merged**